R/bind_marginals.R

```diff
@@ -6,14 +6,13 @@
 #' @param dst_x marginal distribution
 #' @param dst_y marginal distribution
 #' @return A list with copula, dst_x and dst_y objects with their names retained.
-#' The list belongs to cop_dst class, which is a class of bi_dst, which is a class of mult_dst.
+#' The list belongs to cop_dst class, which is a class of bi_dst, which is a class of multi_dst.
 #' @examples
 #' bind_marginals(cop_ig(theta = 1, alpha = 3), mtcars$hp, mtcars$disp)
-#' bind_marginals(cop_ig(theta = 1, alpha=2), rnorm(50, 2), rnorm(50, 15))
+#' bind_marginals(cop_ig(theta = 1, alpha=2), distionary::dst_norm(50, 2), distionary::dst_norm(50, 15))
 #'
 #' @export
 bind_marginals <- function(copula, dst_x, dst_y){
-  bivariate_dst <- list(copula, dst_x, dst_y)
-  names(bivariate_dst) <- c("copula", "dst_x", "dst_y")
+  bivariate_dst <- list(copula = 'copula', dst_x = 'dst_x', dst_y = 'dst_y')
```

Comment:
Need to drop the quotation marks, otherwise this list will be storing strings. Should be 

`list(copula = copula, dst_x = dst_x, dst_y = dst_y)`

(the left-hand-side sets the name of the entry; the right-hand-side gets evaluated and stored in the list)